src/demos/calcite-dismissible-panel.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+
+    <title>Calcite App Components: calcite-dismissible-panel</title>
+
+    <!-- Loads the link and script tags-->
+    <script src="head.js"></script>
+    <style>
+      .panel-container {
+        width: 350px;
+        height: 800px;
+      }
+    </style>
+  </head>
+
+  <body>
+    <nav>
+      <calcite-demo-nav page-id="calcite-dismissible-panel"></calcite-demo-nav>
+    </nav>
+    <main>
+      <section class="example-container">
+        <h1>calcite-dismissible-panel</h1>
+        <p>
+          see the
+          <a href="https://github.com/Esri/calcite-app-components/tree/master/src/components/calcite-dismissible-panel/readme.md">
+            calcite-dismissible-panel readme.</a
+          >
+        </p>
+
+        <div class="panel-container">
+          <calcite-dismissible-panel heading="test">
+            <p>Content</p>
+          </calcite-panel>
```

Comment:
typo on closing tag

Reply:
hey! this isn't ready for review yet

Reply:
oh glad you told me, was about to dive into it :D